### ISSUE TITLE
Red color not working

### ISSUE THREAD

**Issue by Thesandlord** [pravj/geo_pattern#6]
I'm having some issues with the colors being generated.

I am using pure red, #ff0000

When I send in this map as the args:

map[string]string{"phrase": "abc", "generator": "xes", "color": "#ff0000"}

I am getting back this SVG:
![screen shot 2015-03-05 at 6 25 14 pm](https://cloud.githubusercontent.com/assets/8902396/6519129/df34d5b2-c364-11e4-89d1-4b43db1a6783.png)

Raw SVG string:
http://pastebin.com/A4R6mpnh

When I try these two:
map[string]string{"phrase": "abc", "generator": "xes", "color": "#ff2211"}
map[string]string{"phrase": "abc", "generator": "xes", "color": "#aa2211"}

The output image is exactly the same.

Basically, it looks like it is completely ignoring the Red. Blue and Green seem to work fine.

**Comment by Thesandlord** [comment]
Just figured it out, looks like a bug in the code:

Line 110 in pattern.go, rgb.R is divided by 2 instead of multiplied by 105 like everything else.

Will submit a pull request

**Comment by pravj** [comment]
Hey @Thesandlord,

I just woke up and saw the issue email.
It's weird as I didn't face the issue while generating sample patterns.

Awesome that you figured it out quickly, yeah! send a pull request and I'll plug that in. :+1:
